addons/cargo/functions/fnc_startLoadIn.sqf

```diff
@@ -16,15 +16,16 @@
  */
 #include "script_component.hpp"
 
-params ["_player", "_object"];
-TRACE_2("params",_player,_object);
-
-private _vehicle = [_player, _object] call FUNC(findNearestVehicle);
-
-if ((isNull _vehicle) || {_vehicle isKindOf "Cargo_Base_F"}) then {
-    {
-        if ([_object, _x] call FUNC(canLoadItemIn)) exitWith {_vehicle = _x};
-    } forEach (nearestObjects [_player, ["Cargo_base_F", "Land_PaperBox_closed_F"], MAX_LOAD_DISTANCE]);
+params ["_player", "_object", ["_vehicle", objNull]];
+TRACE_3("params",_player,_object,_vehicle);
+
+if (isNull _vehicle) then {
+    _vehicle = [_player, _object] call FUNC(findNearestVehicle);
```

Comment:
Please don't reuse variable names in this way. Name it differently and set it to private.